hmt_escrow_js/ETHInterface.js

```diff
@@ -0,0 +1,146 @@
+require('dotenv').config()
+
+const EscrowABI= require('./ABIs/Escrow')
+const EscrowFactoryABI = require('./ABIs/EscrowFactory')
+const HMTTokenInterfaceABI = require('./ABIs/HMTTokenInterface')
+
+const HDWalletProvider = require("truffle-hdwallet-provider")
+const web3 = require('web3')
+const isUrl = require('is-url')
+
+let HMTOKEN_ADDR = process.env.HMTOKEN_ADDR || "0x4C18A2E51edC5043e9c4B6b0757990A4Ac13797f"
```

Comment:
Just a kinky note, but this is not our current HMTOKEN_ADDR, which is `0x4dCf5ac4509888714dd43A5cCc46d7ab389D9c23`.

Reply:
From your codebase.  Fixing it
```
HMTOKEN_ADDR = Web3.toChecksumAddress( os.getenv("HMTOKEN_ADDR", "0x4C18A2E51edC5043e9c4B6b0757990A4Ac13797f"))
```